tree/ntuple/v7/inc/ROOT/RNTupleFillContext.hxx

```diff
@@ -122,6 +128,15 @@ public:
    /// Return the number of entries filled so far.
    NTupleSize_t GetNEntries() const { return fNEntries; }
 
+   void EnableStagedClusterCommitting(bool val = true)
+   {
+      if (!val && !fStagedClusters.empty()) {
+         throw RException(R__FAIL("cannot disable staged committing with pending clusters"));
+      }
+      fStagedClusterCommitting = val;
+   }
+   bool IsStagedClusterCommittingEnabled() const { return fStagedClusterCommitting; }
```

Comment:
Side note, not for this PR: we should use a consistent naming in RNTuple for option setting. E.g. decide on `EnableX`, `SetEnableX`, etc.

Reply:
I agree consistency would be good. My model was that the active `Enable` without `Set` performs an action and changes the object's state, whereas `RNTupleWriteOptions::SetEnablePageChecksums` only sets the value which has an effect when the reader is created...